libcontainer/cgroups/fs/apply_raw.go

```diff
@@ -106,6 +106,13 @@ func (m *Manager) Apply(pid int) (err error) {
 		return nil
 	}
 
+	var c = m.Cgroups
+
+	if c.Paths != nil {
+		m.Paths = c.Paths
+		return cgroups.EnterPid(m.Paths, pid)
+	}
+
```

Comment:
Sorry didn't make it clearer, I meant maybe we should do something like (This is what we did in `Apply()`):
```
for name, path := range c.Paths {
        p, err := d.path(name)
        if err != nil {
                if cgroups.IsNotFound(err) {
                        continue
                }
                return err
        }
        paths[name] = path
}
m.Paths = paths

return cgroups.EnterPid(m.Paths, pid)
```

Doing this because `m.Path` will be used by `GetPaths`, `GetStats`, etc. It'll be easier for them if it holds only valid paths.